src/app/common/components/TableEmptyState.tsx

```diff
@@ -10,6 +10,7 @@ import {
   Button,
 } from '@patternfly/react-core';
 import { SearchIcon } from '@patternfly/react-icons';
+import spacing from '@patternfly/react-styles/css/utilities/Spacing/spacing';
```

Comment:
Same comment here, changes on this file do not seem relevant to this PR.

Reply:
Oops! You're right. I was originally using TableEmptyState in HooksForm but decided it was easier to just directly use an EmptyState than make this component more complicated. Meant to revert this, I'll do so.